Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -1885,111 +1939,228 @@ public void getShortUrl(String channel, String feature, String stage, JSONObject
      *                 the link that will be passed into a new app session when clicked
      * @param callback A {@link BranchLinkCreateListener} callback instance that will trigger
      *                 actions defined therein upon receipt of a response to a create link request.
+     * @param  async   A {@link Boolean} whose value is true if the link should be created asynchronously.
```

Comment:
Don't need this; the presence of a callback is sufficient to determine that it's async.